### PR TITLE
fix: author of single article 

### PR DESCRIPTION
**Pull request by xiaopengli89**
Hi,

This is a pretty nice theme, I like it! I just fix the author of single article.

### CODE REVIEW

File: layouts/partials/render_single_article.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {{ if or (.Date) (ne .Params.Kind "page") }}
 <div class="date">
     <time pubdate="{{ .Date.Format "2006-01-02" }}">{{ .Date.Format "January 2, 2006" }}</time>
-    <span class="author">by Pravin Paratey</span>
+    <span class="author">by {{ .Params.author }}</span>
```
Comment (pravin) [inline layouts/partials/render_single_article.html:4]: Oops! Thanks :)

File: layouts/partials/menu.html [hunk 2]
```diff
@@ -68,7 +68,7 @@
         </li>
         {{ end }}
         <li class="pure-menu-item">
-            <a href="/atom.xml" title="Atom Feed" class="pure-menu-link">
+            <a href="/posts/index.xml" title="Atom Feed" class="pure-menu-link">
```
Comment (pravin) [inline layouts/partials/menu.html:71]: Good catch!

File: layouts/partials/custom_footer.html [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.20.0/prism.min.js"></script>
```
Comment (pravin) [inline layouts/partials/custom_footer.html:1]: Are you able to leave this bit out of the PR please as this is custom to how you are using it?

**Review by pravin — CHANGES_REQUESTED** [review]
Sorry for the late review! Looks good and thanks for catching a few errors! Thanks for the CSS changes too.

Can you resubmit the PR without the prism changes to custom_header / custom_footer please?

**Comment by xiaopengli89** [comment]
Ok, I resubmit the PR.

**Pull request closed without merging**